xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/resources/document-formats.js

```diff
@@ -0,0 +1,191 @@
+[
```

Comment:
Small note: would need a license header here.

Reply:
Indeed I'll add it.